tests/e2e/apps_test.go

```diff
@@ -446,6 +447,13 @@ var _ = Describe("Apps", func() {
 					SetResult(&result).
 					Get("/v3/apps/" + appGUID + "/env")
 			}).Should(HaveRestyStatusCode(http.StatusOK), BeNil())
+
+			Eventually(func(g gomega.Gomega) {
```

Comment:
Should this block replace the original one on line 445 as it covers the HTTP status code check?

Also, you are reimporting the `gomega` package, which is already dot-imported.